python/3.11/asynctio_unix_events.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/Lib/asyncio/unix_events.py b/Lib/asyncio/unix_events.py
+index f34a5b4b44..b1d0f1e61e 100644
+--- a/Lib/asyncio/unix_events.py
++++ b/Lib/asyncio/unix_events.py
+@@ -369,6 +369,11 @@ class _UnixSelectorEventLoop(selector_events.BaseSelectorEventLoop):

```

Comment:
These line numbers no longer match. Should this not be 391 instead of 369?
https://github.com/python/cpython/blob/3.9/Lib/asyncio/unix_events.py#L369
https://github.com/python/cpython/blob/3.11/Lib/asyncio/unix_events.py#L391

Reply:
Patch will deal with that :)